examples/tracing/otlp/README.md

```diff
@@ -65,3 +65,37 @@ service:
 
 1. Open the Datadog traces UI under the menu item on the left `APM` then `Traces`
 1. You should see a list of traces to explore
+
+### Clickhouse
+
+#### Requirements
+
+1. [Clickhouse CLI](https://clickhouse.com/docs/en/install)
```

Comment:
You also could use `docker compose exec clickhouse clickhouse client` without installing the client on host machine.

Reply:
@erka Indeed. Yeah that would be better for ppl who do not want it on their machine.

Reply:
I think i'll remove that part, because on port `8123` you get a UI which you can query from anyway.